python/ray/services.py

```diff
@@ -941,6 +943,10 @@ def start_raylet(redis_address,
             to.
         worker_path (str): The path of the script to use when the local
             scheduler starts up new workers.
+        use_valgrind (bool): True if the plasma store should be started inside
+            of valgrind. If this is True, use_profiler must be False.
+        use_profiler (bool): True if the plasma store should be started inside
+            a profiler. If this is True, use_valgrind must be False.
```

Comment:
These comments need to be updated. `plasma store` -> `raylet`

Reply:
Thanks, did it!